docs/trulens_eval/clear_trulens_module.ipynb

```diff
@@ -0,0 +1,24 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "\n",
+    "for module in list(sys.modules.keys()):\n",
```

Comment:
explanation needed, why is this needed?

Reply:
Added a comment in the code - this solves some state-induced errors across notebooks [here](https://dev.azure.com/truera/trulens/_build/results?buildId=9485&view=logs&j=7b1c58dd-fec1-5ac7-6a25-5bff49a02517&t=96cb52ed-0cad-5d6c-9e27-b912252d0fb1&l=41)